scripts/update/Updater.php

```diff
@@ -430,6 +430,6 @@ public function update($initialVersion)
             $this->setVersion('11.3.2');
         }
 
-        $this->skip('11.3.2', '11.4.2');
+        $this->skip('11.3.2', '12.0.0');
```

Comment:
I think the Updater should care about updating the table schema. What do you think?

Reply:
Yes, but this will need a bigger work, since we'd have to deal with existing tao instances. We had a small talk with Joel about it last night and this won't be merged as soon...